include/pagmo/algorithm.hpp

```diff
@@ -542,8 +552,13 @@ class algorithm
      * as the UDA used during construction (after removal of cv and reference qualifiers), this method will
      * return \p nullptr.
      *
-     * **NOTE** The returned value is a raw non-owning pointer: the lifetime of the pointee is tied to the lifetime
-     * of \p this and \p delete must never be called on the pointer.
+     * \verbatim embed:rst:leading-asterisk
+     * .. note::
+     *
+     *    The returned value is a raw non-owning pointer: the lifetime of the pointee is tied to the lifetime
```

Comment:
Since we are in a sphinx syntax block, the "\p" syntax should become double backticks.

There's a bunch of other instances of this, not gonna repeat :)